lib/ts/controllers/s-tooltip.ts

```diff
@@ -50,6 +51,23 @@ namespace Stacks {
             super.show(dispatcher);
         }
 
+        /**
+         * Sets up a tooltip popover show after a delay.
+         */
+        scheduleShow(dispatcher: Event | Element | null = null) {
+            this.activeTimeout = setTimeout(() => this.show(dispatcher), 300);
+        }
+
+        /**
+         * Cancels the scheduled tooltip popover display and hides it if already displayed
+         */
+        hide(dispatcher: Event | Element | null = null) {
+            clearTimeout(this.activeTimeout);
```

Comment:
Sanity checking: [MDN](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout#Notes) states that it is 100% ok to pass invalid ids here (such as `null` if `hide` is called before `scheduleShow`).

Optional: Personal preference. Commit if desired.

```suggestion
            window.clearTimeout(this.activeTimeout);
```